uiowa/uiowa01
# Feature: Remove duplicate styles from admin css

Work on #292.

# How to test

1. run `drush @theming.local si collegiate`
2. create a basic page with list items and nested list items
3. save page and verify that list styles are being implemented
